packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -371,6 +376,48 @@ export class AndroidEmulatorDevice extends DeviceBase {
     ]);
   }
 
+  async mirrorNativeLogsDispose() {
+    this.nativeLogsOutputChannel?.dispose();
+    this.nativeLogsProcess?.kill();
+  }
```

Comment:
maybe not worth extracting to separate method

Reply:
Changed in: https://github.com/software-mansion/radon-ide/pull/734/commits/0fe9fda34e73e2eedf7acbc3f3aa70434ca395b4